travis-ci/vmtest/run_selftests.sh

```diff
@@ -26,25 +26,31 @@ test_progs() {
 		echo "test_progs:$?" >> "${STATUS_FILE}"
 		foldable end test_progs
 	fi
```

Comment:
I assume it is correct, but will double check. Does it mean that we execute only no_alu32 tests each time? All other tests are being executed depending on the kernel version?

Reply:
E.g. on latest we execute all the tests, on 5.* we execute prog_tests and no_alu_32, and on 4.9.0 we execute only no_alu32

Reply:
Yep, that seems to be the case.

Reply:
That behavior is unchanged from earlier. E.g., check the selftest runs on https://github.com/libbpf/libbpf/pull/546.

Reply:
oh, I know, it is just something that caught my eye